app/controllers/notification.py

```diff
@@ -30,7 +126,7 @@ class NotificationController(Controller):
     async def get_notification_key(self) -> str:
         return env.VAPID_APPLICATION_SERVER_KEY
 
-    @post("/api/v1/notifications", return_dto=None)
+    @post("/api/v1/notifications")
```

Comment:
Shouldn't this only be available to authenticated users?
Or are you planning on doing that in a future PR, requiring a pro-connection from `ami_admin`?
If it'll be done in a future PR, do we already have an open issue for that?

Reply:
This endpoint has to be deleted once mobile app uses the authenticated endpoint.
At this moment, we also have to create an "internal" endpoint for rvo and ami_admin, with a way to authenticate the request. Mays be with a API_KEY_TOKEN ?

Reply:
cf #303